README.md

```diff
@@ -1,5 +1,39 @@
-# Firebase Functions Python SDK
+# Cloud Functions for Firebase Python SDK (Public Preview)
 
-Welcome GitHub lurker! This repository is a work in progress. It will eventually let users create Cloud Functions for Firebase in Python. This feature is under very early development and not ready for testing. When it is ready, you can get first access by signing up for the [Firebase Alpha Program](https://services.google.cn/fb/forms/firebasealphaprogram/).
+The [`firebase-functions`](https://pypi.org/project/firebase-functions/) package provides an SDK for defining Cloud Functions for Firebase in Python.
 
-While you wait for us to get this feature ready for the public, we'd really appreciate you not blowing this up. Let's keep things on the down low for now. 🤫
+Cloud Functions provides hosted, private, and scalable environment where you can run server code. The Firebase SDK for Cloud Functions integrates the Firebase platform by letting you write code that responds to events and invokes functionality exposed by other Firebase features.
+
+## Learn more
+
+Learn more about the Firebase SDK for Cloud Functions in the [Firebase documentation](https://firebase.google.com/docs/functions/) or [check out our samples](https://github.com/firebase/functions-samples).
+
+Here are some resources to get help:
+
+- Start with the quickstart: https://firebase.google.com/docs/functions/write-firebase-functions
+- Go through the guide: https://firebase.google.com/docs/functions/
+- Read the full API reference: https://firebase.google.com/docs/reference/functions/
+- Browse some examples: https://github.com/firebase/functions-samples
+
+If the official documentation doesn't help, try asking through our official support channels: https://firebase.google.com/support/
+
+## Usage
+
+```python
+# functions/main.py
+from firebase_functions import db_fn
+from notify_users import api
+
+@db_fn.on_value_created(reference="/posts/{post_id}")
+def new_post(event):
+    print(f"Received new post with ID: {event.params.get('post_id')}")
```

Comment:
Function names with underscores are not allowed if I remember correctly. But maybe that changed?

Reply:
Also see https://github.com/firebase/firebase-tools/pull/5612. I really hope to be wrong here as this would be very idiomatic.

Reply:
It did change! A recent update released in the cli: https://github.com/firebase/firebase-tools/releases/tag/v11.29.0

Reply:
Oh awesome! Sorry for the noise then. I will close that PR as well.